app/src/app/[lng]/organization/[id]/account-settings/project/index.tsx

```diff
@@ -0,0 +1,851 @@
+              >
+                <Icon as={LuChevronDown} color="currentColor" boxSize={4} />
+              </Accordion.ItemIndicator>
+            </Button>
+          </AccordionItemTrigger>
+          <AccordionItemContent padding="0px" pb={4}>
+            {item.cities.length === 0 ? (
+              <Text fontSize="body.lg" fontWeight={600} color="content.primary">
+                {t("no-cities")}
+              </Text>
+            ) : (
+              <Tabs.Root
+                display="flex"
+                mt="12px"
+                flexDirection="row"
+                variant="subtle"
+                w="full"
+                gap="12px"
+                value={selectedProjectId[0]} // Use selectedProjectId[0] as value to indicate selected project
+                onValueChange={(val) => setSelectedCity(val.value)}
+              >
+                <Tabs.List
+                  w="full"
+                  display="flex"
+                  flexDirection="column"
+                  gap="12px"
+                >
+                  {item.cities.map((city) => (
+                    <Tabs.Trigger
+                      key={city.cityId}
+                      value={city.cityId}
+                      fontFamily="heading"
+                      justifyContent={"left"}
+                      letterSpacing={"wide"}
+                      color="content.secondary"
+                      lineHeight="20px"
+                      fontStyle="normal"
+                      fontSize="label.lg"
+                      minH="52px"
+                      w="full"
+                      _selected={{
+                        color: "content.link",
+                        fontSize: "label.lg",
+                        fontWeight: "medium",
+                        backgroundColor: "background.neutral",
+                        borderRadius: "8px",
+                        borderWidth: "1px",
+                        borderStyle: "solid",
+                        borderColor: "content.link",
+                      }}
+                    >
+                      {city.name}
+                    </Tabs.Trigger>
+                  ))}
+                </Tabs.List>
+              </Tabs.Root>
+            )}
+          </AccordionItemContent>
+        </AccordionItem>
+      ))}
+    </AccordionRoot>
+  </Box>
+);
+
+interface ProjectDetailsProps {
+  t: Function;
+  lng: string;
+  router: any;
+  selectedCity: string | null;
+  selectedProjectData: ProjectWithCities | null | undefined;
+  selectedCityData: any; // Define a more specific type if possible
+  organizationName?: string;
+  projectUsers: any[]; // Define a more specific type if possible
+  userList: any[]; // Define a more specific type if possible
+  isLoadingProjectUsers: boolean;
+  tabValue: string;
+  setTabValue: (value: string) => void;
+  setSelectedCity: (value: string | null) => void;
+}
+
+const ProjectDetails: React.FC<ProjectDetailsProps> = ({
```

Comment:
There are a lot of components in this file, might be good to extract them to their own files and re-export them from the index.tsx file?